server/src/models/ip/ip.dto.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  IsNotEmpty,
+  IsString,
+  IsIP,
+} from 'class-validator';
+
+export class AddIPDto {
+  @IsIP()
+  @IsNotEmpty()
+  ip: string;
+
+  @IsNotEmpty()
```

Comment:
Should we force `description` to be a part of the request.

Reply:
yes! good catch